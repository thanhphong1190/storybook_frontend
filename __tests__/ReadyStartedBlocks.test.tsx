import renderer from 'react-test-renderer'
import {
  ReadyStartedBlocks,
  ReadyStartedBlocksProps
} from '../src/components/ready-started-blocks/ReadyStartedBlocks'

const props: ReadyStartedBlocksProps = {
  blocks: [
    {
      text: 'Download<br/>E-brochure',
      imgURL: '/example/Ready-started-blocks/icon-brochure.png',
      color: 'blue-dark',
      link: '#'
    },
    {
      text: 'Find a<br/> Franchise',
      imgURL: '/example/Ready-started-blocks/icon-find.png',
      color: 'blue',
      link: '#'
    },
    {
      text: 'Enquire Now',
      imgURL: '/example/Ready-started-blocks/icon-enquire.png',
      color: 'orange',
      link: '#'
    }
  ]
}

it('ReadyStartedBlocks renders correctly', () => {
  const tree = renderer.create(<ReadyStartedBlocks {...props} />).toJSON()
  expect(tree).toMatchSnapshot()
})
