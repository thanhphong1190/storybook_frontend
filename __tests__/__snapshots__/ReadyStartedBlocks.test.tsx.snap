// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReadyStartedBlocks renders correctly 1`] = `
<section
  className="ReadyStartedBlocksstyles__Section-sc-14xu6dg-0 gWJzDe"
>
  <div
    className="ReadyStartedBlocksstyles__Container-sc-14xu6dg-1 igEEQg"
  >
    <div
      className="ReadyStartedBlocksstyles__BlocksWrapper-sc-14xu6dg-2 jOFlxR"
    >
      <div
        className="ReadyStartedBlocksstyles__Block-sc-14xu6dg-3 gjTnzi"
        color="blue-dark"
      >
        <a
          className="ReadyStartedBlocksstyles__LinkWrapper-sc-14xu6dg-4 gntxaK"
          href="#"
        >
          <div
            className="ReadyStartedBlocksstyles__ImgWrapper-sc-14xu6dg-5 jOfCOE"
          >
            <img
              className="ReadyStartedBlocksstyles__IMG-sc-14xu6dg-6 gNkMXj"
              src="/example/Ready-started-blocks/icon-brochure.png"
            />
          </div>
          <span
            className="ReadyStartedBlocksstyles__Text-sc-14xu6dg-7 cxulkE"
            dangerouslySetInnerHTML={
              Object {
                "__html": "Download<br/>E-brochure",
              }
            }
          />
        </a>
      </div>
      <div
        className="ReadyStartedBlocksstyles__Block-sc-14xu6dg-3 cNjtRa"
        color="blue"
      >
        <a
          className="ReadyStartedBlocksstyles__LinkWrapper-sc-14xu6dg-4 gntxaK"
          href="#"
        >
          <div
            className="ReadyStartedBlocksstyles__ImgWrapper-sc-14xu6dg-5 jOfCOE"
          >
            <img
              className="ReadyStartedBlocksstyles__IMG-sc-14xu6dg-6 gNkMXj"
              src="/example/Ready-started-blocks/icon-find.png"
            />
          </div>
          <span
            className="ReadyStartedBlocksstyles__Text-sc-14xu6dg-7 cxulkE"
            dangerouslySetInnerHTML={
              Object {
                "__html": "Find a<br/> Franchise",
              }
            }
          />
        </a>
      </div>
      <div
        className="ReadyStartedBlocksstyles__Block-sc-14xu6dg-3 eCIcVp"
        color="orange"
      >
        <a
          className="ReadyStartedBlocksstyles__LinkWrapper-sc-14xu6dg-4 gntxaK"
          href="#"
        >
          <div
            className="ReadyStartedBlocksstyles__ImgWrapper-sc-14xu6dg-5 jOfCOE"
          >
            <img
              className="ReadyStartedBlocksstyles__IMG-sc-14xu6dg-6 gNkMXj"
              src="/example/Ready-started-blocks/icon-enquire.png"
            />
          </div>
          <span
            className="ReadyStartedBlocksstyles__Text-sc-14xu6dg-7 cxulkE"
            dangerouslySetInnerHTML={
              Object {
                "__html": "Enquire Now",
              }
            }
          />
        </a>
      </div>
    </div>
  </div>
</section>
`;
